kong/clustering/rpc/manager.lua

```diff
@@ -289,7 +289,7 @@ function _M:handle_websocket()
   self:_add_socket(s, rpc_capabilities)
 
   -- store DP's ip addr
-  self.client_ips[node_id] = ngx_var.remote_addr
+  self.client_info[node_id] =  { ip = ngx_var.remote_addr, version = kong_version, }
```

Comment:
This doesn't seem right. Are we storing DP's version or CP's?

Reply:
It is dp's version, see data_plane.lua (sync.v1).